miner/worker_test.go

```diff
@@ -171,9 +168,6 @@ func newTestWorkerBackend(t *testing.T, chainConfig *params.ChainConfig, engine
 	if n > 0 {
 		parent = chain.GetBlockByHash(chain.CurrentBlock().ParentHash())
 	}
-	blocks, _ := core.GenerateChain(chainConfig, parent, engine, db, 1, func(i int, gen *core.BlockGen) {
```

Comment:
Should this be removed?

Reply:
My interpretation is that `blocks` is only used to pass an uncle block to the struct below. Since I've deleted that field from the struct, go gives me an error because `blocks` is defined and not used. Is there something I might be missing?